Examples/Examples/BookmarksExampleView.swift

```diff
@@ -31,44 +29,38 @@ struct BookmarksExampleView: View {
     @State var viewpoint: Viewpoint?
     
     var body: some View {
-        MapView(map: dataModel.map, viewpoint: viewpoint)
-            .onViewpointChanged(kind: .centerAndScale) {
-                viewpoint = $0
-            }
-            .toolbar {
-                ToolbarItem(placement: .primaryAction) {
-                    Button {
-                        showingBookmarks.toggle()
-                    } label: {
-                        Label(
-                            "Show Bookmarks",
-                            systemImage: "bookmark"
-                        )
-                    }
-                    .popover(isPresented: $showingBookmarks) {
-                        // Display the `Bookmarks` components with a pre-defined
-                        // list of bookmarks. Passing in a `Viewpoint` binding
-                        // will allow the `Bookmarks` component to handle
-                        // bookmark selection.
-                        Bookmarks(
-                            isPresented: $showingBookmarks,
-                            mapOrScene: dataModel.map,
-                            viewpoint: $viewpoint
-                        )
-                        
-                        // Display the `Bookmarks` component with the list of
-                        // bookmarks in a map.
-//                        Bookmarks(
-//                            isPresented: $showingBookmarks,
-//                            mapOrScene: map
-//                        )
```

Comment:
This would've caused a build error if it were to be uncommented by a user since the map was moved to `dataModel`.